felix/bpf-gpl/qos.h

```diff
@@ -111,4 +112,44 @@ static CALI_BPF_INLINE int enforce_packet_rate_qos(struct cali_tc_ctx *ctx)
 	return TC_ACT_SHOT;
 }
 
+static CALI_BPF_INLINE int set_dscp(struct cali_tc_ctx *ctx)
+{
+#if (CALI_F_FROM_WEP || CALI_F_TO_HEP)
+	// TODO (mazdak): set DSCP only if traffic is leaving cluster
+	if (EGRESS_DSCP < 0) {
+		return TC_ACT_UNSPEC;
+	}
```

Comment:
Would it make sense to take it out from this function? Could -1 be technically a valid value? (guess not since all valid values are 6bits only). Perhaps a flag that this field is set?